File: src/Domain/Entities/Veterinario.cs
using TrazabilidadIberica.Domain.Common;

namespace TrazabilidadIberica.Domain.Entities;

public class Veterinario : BaseEntity
{
    public required string NombreCompleto { get; set; }
    public required string NumColegiado { get; set; }
    public string? Telefono { get; set; }
    public string? Email { get; set; }

    public ICollection<TratamientoVeterinario> Tratamientos { get; set; } = new List<TratamientoVeterinario>();
}

File: frontend/src/app/core/models/veterinario.ts
export interface Veterinario {
  id: string;
  nombreCompleto: string;
  numeroColegiado: string;
  telefono: string | null;
  email: string | null;
  createdOffline: boolean;
  clientId: string | null;
  createdAt: string;
  updatedAt: string;
  deletedAt: string | null;
}
